test/dlc_tests/ec2/test_efa.py

```diff
@@ -76,6 +78,12 @@ def test_pytorch_efa(
     :param region: str Region in which EFA-enabled instances are launched
     :param gpu_only: pytest fixture to limit test only to GPU DLCs
     """
+
+    # NOTE: Skip PT1.13 autopatching
```

Comment:
putting skip condition inside the test function is not a good idea, because test setup will still launch instances. Instead, you can probably use an existing fixture like pt21_and_above_only